pyproject.toml

```diff
@@ -12,7 +12,7 @@ requests = "^2.25.1"
 # singer-sdk = "^0.4.9"
 # For local SDK dev:
 # singer-sdk = {path = "../singer-sdk", develop = true}
-singer-sdk = {git = "https://github.com/oviohub/meltano-sdk.git", rev = "8e505a074c1bca31bdc7bb3e2c8102fe95bcbe05"}
```

Comment:
We don't have a pinned version that fits our need yet?

Reply:
It would be nice to move back to a version of the upstream repo

Reply:
I'm planning to go back to a normal version once the sdk PR is merged, but this allows us to test this code early.

Reply:
Should we wait to merge?

Reply:
Yes, I would wait until the sdk is officially supporting the cost logging logic, then we can update and merge this. But for now, this branch allows testing and seeing if the result is useful/usable.

Reply:
For reference - https://github.com/meltano/sdk/pull/704

Reply:
@laurentS the PR above has been merged :)